packages/synthetix-main/contracts/mixins/CollateralMixin.sol

```diff
@@ -23,7 +23,7 @@ contract CollateralMixin is CollateralStorage, PoolVaultStorage {
         _;
     }
 
-    function _getCollateralValue(address collateralType) internal view returns (uint) {
+    function _collateralValue(address collateralType) internal view returns (uint) {
```

Comment:
Shouldn't we also follow the `get<Thing>` pattern for internal functions? I.e. use "get" for all view functions, internal or external.